.github/workflows/build_and_test.yaml

```diff
@@ -147,6 +150,14 @@ jobs:
               run: ${{ matrix.prepare_cmd }}
               if: matrix.prepare_cmd
 
+            # This is required for Playwright testing
```

Comment:
Yes, but what does it actually do? Is it *re-enabling* a fuse that was previously disabled by `electron-builder`? It would be good to say so.